src/object.js

```diff
@@ -37,5 +37,62 @@ _.extend(Marionette.Object.prototype, Backbone.Events, {
   bindEntityEvents: Marionette.proxyBindEntityEvents,
 
   // Proxy `unbindEntityEvents` to enable unbinding view's events from another entity.
-  unbindEntityEvents: Marionette.proxyUnbindEntityEvents
+  unbindEntityEvents: Marionette.proxyUnbindEntityEvents,
+
+  //Proxy Radio message handling to enable declarative interactions with radio channels
+  _radioTypes: {
+    'radioEvents' : {
+      startMethod: 'on',
+      stopMethod: 'off'
+    },
+    'radioCommands' : {
+      startMethod: 'comply',
+      stopMethod: 'stopComplying'
+    },
+    'radioRequests' : {
+      startMethod: 'reply',
+      stopMethod: 'stopReplying'
+    }
+  },
+  //store the radio channels being used
+  _radioChannels: [],
+
+  _proxyRadioHandlers: function() {
+    this._unproxyRadioEvents();
+    _.each(this._radioTypes, function(commands, radioType) {
+      var hash = _.result(this, radioType);
+      if (!hash) {
+        return;
+      }
+      _.each(hash, function(handler, radioMessage) {
+        if (!_.isFunction(handler)) {
+          handler = this[handler];
+        }
+        if (!handler) {
+          return;
+        }
+        var messageComponents = radioMessage.split(' '),
+          channel = messageComponents[0],
+          messageName = messageComponents[1];
+        this._proxyRadioHandler(channel, radioType, messageName, handler);
+
+      }, this);
+  }, this);
+},
+
+  _proxyRadioHandler: function(channel, radioType, messageName, handler) {
+    var method = this._radioTypes[radioType].startMethod;
+    Backbone.Radio[method](channel, messageName, handler, this);
+  },
+
+  _unproxyRadioEvents: function() {
+    _.each(this._radioChannels, function(channel) {
+      _.each(this._radioTypes,function(commands) {
+        Backbone.Radio[commands.stopMethod](channel,this);
```

Comment:
Passing in `this` as the second argument will prevent unbinding I believe. The second argument needs to be the callback and then the context. Obviously, `listenTo/stopListening` and the forthcoming command/reply counterparts would be preferred here. Otherwise, we'll need to track the callbacks we add, which is really just reimplementing `listenTo`-type behavior.